force-app/main/default/objects/ProgramEngagement__c/validationRules/PROGRAM_MATCHES_COHORT_PROGRAM.validationRule-meta.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>PROGRAM_MATCHES_COHORT_PROGRAM</fullName>
+    <active>true</active>
+    <description>The Program must match the Program Cohort&#39;s Program when Program Cohort is populated.</description>
+    <errorConditionFormula>AND(
+!ISBLANK(Program__c),
+!ISBLANK(ProgramCohort__c),
+ProgramCohort__r.Program__c != Program__c
+)</errorConditionFormula>
+    <errorMessage>Select a Program Cohort that matches the Program.</errorMessage>
```

Comment:
The Work Item is requesting a filter, assuming we changed to a validation rule but wanted to confirm? Also, our test data is now failing. Should that be a separate work item or handled here?
"Error on record with id 55: FIELD_CUSTOM_VALIDATION_EXCEPTION:Select a Program Cohort that matches the Program.:--"

Reply:
Oh shoot that should be fixed here. Let me tackle that this morning. Thanks for the review.

Reply:
Done!